cmd/plugin.go

```diff
@@ -864,3 +869,28 @@ func getConnectionState(ctx context.Context) (steampipeconfig.ConnectionStateMap
 
 	return connectionStateMap, res
 }
+
+func cleanupOldTmpDirs() {
+	const tmpDirAgeThreshold = 24 * time.Hour
+	tmpDirs, err := files.ListFiles(filepaths.EnsurePluginDir(), &files.ListOptions{
+		Include: []string{"tmp-*"},
+		Flags:   files.AllRecursive,
```

Comment:
use flag `files.DirectoriesRecursive` since we only are interested in dirs
```suggestion
		Flags:   files.DirectoriesRecursive,
```